drools-wb-webapp/pom.xml

```diff
@@ -1262,26 +1275,191 @@
       <artifactId>org.eclipse.sisu.inject</artifactId>
       <scope>runtime</scope>
     </dependency>
+
+    <!-- Included dependencies because previously where cross-provided by other components. -->
```

Comment:
Just to be aware of the proper naming: what you call cross-provided is what maven officially calls "transitive dependencies".